docker-compose.yml

```diff
@@ -1,6 +1,36 @@
-version: '3'
+version: "3"
 
 services:
+  database:
+    image: quay.io/azavea/postgis:3-postgres12.4-slim
+    environment:
+      - POSTGRES_USER=franklin
+      - POSTGRES_PASSWORD=franklin
+      - POSTGRES_DB=franklin
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "franklin"]
+      interval: 3s
+      timeout: 3s
+      retries: 3
+      start_period: 5s
+  franklin:
+    image: quay.io/azavea/franklin:c93f2ff
+    depends_on:
+      database:
+        condition: service_healthy
+    command: ["serve", "--with-transactions"]
+    environment:
+      - ENVIRONMENT=development
+      - DB_HOST=database.service.internal
+      - DB_NAME=franklin
+      - DB_USER=franklin
+      - DB_PASSWORD=franklin
+      - AWS_PROFILE
+      - AWS_REGION
+    links:
+      - database:database.service.internal
```

Comment:
Links are now a [legacy feature](https://docs.docker.com/network/links/). I would consider removing this and referencing the service via `DB_HOST=database`.